### PR TITLE
Add pseudo-transactions

### PR DESCRIPTION
**Pull request by mukulljangid**
Add pseudo-transactions:

- SetFee
- EnableAmendment
- UNLModify

Add tests.

### CODE REVIEW

File: xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/TransactionType.java [hunk 1]
```diff
@@ -37,6 +37,11 @@
    */
   DEPOSIT_PRE_AUTH("DepositPreauth"),
 
+  /**
+   * The {@link TransactionType} for the {@link EnableAmendment} transaction.
+   */
+  ENABLE_AMENDMENT("EnableAmendment"),
```
Comment (sappenin) [inline xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/TransactionType.java:43]: Can you add Unit tests that assert the capitalization and contents of each value in this class (i.e., in `TransactionType`)?

File: xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/EnableAmendmentTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+import org.xrpl.xrpl4j.model.client.common.LedgerIndex;
+
+import java.util.Optional;
+
+public class EnableAmendmentTest {
+
+  @Test
+  public void testJson() throws JsonProcessingException, JSONException {
```
Comment (sappenin) [inline xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/EnableAmendmentTest.java:17]: This test doesn't appear to be testing any JSON.  Let's rename the test to something clearer, like this:

```suggestion
  public void testBuilder() throws JsonProcessingException, JSONException {
```

File: xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/SetFeeTest.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+import org.xrpl.xrpl4j.model.client.common.LedgerIndex;
+
+import java.util.Optional;
+
+public class SetFeeTest {
+
+  @Test
+  public void testJson() throws JsonProcessingException, JSONException {
```
Comment (sappenin) [inline xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/SetFeeTest.java:17]: This test doesn't appear to be testing any JSON.  Let's rename the test to something clearer, like this:

```suggestion
  public void testBuilder() throws JsonProcessingException, JSONException {
```

**Review by sappenin — CHANGES_REQUESTED** [review]
All looks good, just one request around testing.

**Review by nkramer44 — APPROVED** [review]
This looks great. Thanks for thinking through this one and nice job figuring out the serializer stuff!

**Comment by mukulljangid** [comment]
@sappenin @nkramer44 Also, this would help us in future, when we introduce FeeSettings ledger object. FeeSettings also has the field, BaseFee, which is also expected to be in hex, so this serializer change will help us there in future.

**Review by sappenin — APPROVED** [review]
LGTM!

**Pull request merged**